config/oauth/idp.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: cluster
 spec:
   identityProviders:
-  - mappingMethod: lookup
+  - mappingMethod: ${MAPPING}
```

Comment:
Are you parameterizing it for local testing? We should never use anything but "lookup" in our stage and prod env.
If it's for local development then fine. But let's mention it in readme.

Reply:
yes, the default value remains `lookup` as you can see in https://github.com/codeready-toolchain/toolchain-infra/pull/34/files#diff-3c99f4ffee6524f96786b5c5a37a00a16a3a5c7ba1b44975e251578a31a3a632R14-R17
and yes, I wanted to use the `claim` mapping method for local testing. I'll mention that in the README ;)

Reply:
done in ad952ea